app/controllers/articles_controller.rb

```diff
@@ -46,6 +46,17 @@ def destroy
     redirect_to articles_url, notice: 'Article was successfully destroyed.'
   end
 
+  def like
+    @article = Article.find(params[:id])
```

Comment:
the article can already be set by `set_article` method in this controller.

Reply:
to keep the CURD style, we can create a new controller similar to commentable and do a similar inheritance like that. So we can reuse this to like other objects other than Articles.